doc/classes/RichTextEffect.xml

```diff
@@ -1,10 +1,20 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="RichTextEffect" inherits="Resource" category="Core" version="3.2">
 	<brief_description>
+		A custom effect for use with [RichTextLabel].
 	</brief_description>
 	<description>
+		A custom effect for use with [RichTextLabel].
+		[b]Note:[/b] For a RichTextEffect to be usable, a BBCode tag must be defined as a member variable called [code]bbcode[/code] in the script.
+		[codeblock]
+		# The RichTextEffect will be usable like this: `[example]Some text[/example]`
+		var bbcode = "example"
+		[/codeblock]
+		[b]Note:[/b] As soon as a [RichTextLabel] contains at least one RichTextEffect, it will continuously process the effect unless the project is paused. This may impact battery life negatively.
```

Comment:
You can also use `[RichTextEffect]`, as mentioned in a previous PR it will automatically convert it to inline code markup when it's the documented class.